src/meta/src/stream/meta.rs

```diff
@@ -90,11 +90,12 @@ where
         Ok(map.values().cloned().collect())
     }
 
-    pub async fn update_table_fragments(&self, table_fragment: TableFragments) -> Result<()> {
+    pub async fn update_table_fragments(&self, mut table_fragment: TableFragments) -> Result<()> {
         let map = &mut self.core.write().await.table_fragments;
 
         match map.entry(table_fragment.table_id()) {
             Entry::Occupied(mut entry) => {
+                table_fragment.seq = entry.get().seq + 1;
```

Comment:
I think `seq` should be updated by outer logic (where vnode mapping of the fragment is reassigned), instead of updated in `FragmentManager`. Here we should only focus on persisting the fragment.

Reply:
I don't think we should care for time complexity too much here, since updating a fragment happens at a very low frequency. Also, a fragment might be updated for other reasons, which has nothing to do with vnode mappings, where we shouldn't update `seq` at all.

Reply:
> I don't think we should care for time complexity too much here, since updating a fragment happens at a very low frequency. Also, a fragment might be updated for other reasons, which has nothing to do with vnode mappings, where we shouldn't update `seq` at all.

but it seems that there is no outer logic currently

Reply:
> but it seems that there is no outer logic currently

Yes. 🤣 So I prefer to leave the function untouched. Someone who writes the logic of updating fragment vnode mapping in the future should be responsible for increasing `seq` as well.

Reply:
> > but it seems that there is no outer logic currently
> 
> Yes. 🤣 So I prefer to leave the function untouched now. Someone who writes the logic of updating fragment vnode mapping in the future should be responsible for increasing `seq` as well.

ok